### ISSUE TITLE
Giant Rockz can only be broken when clicking on a part of it that is accessible by the selected Grunt

### ISSUE THREAD

**Issue by n1ght4ngel19** [n1ght4ngel19/gruntz-unityverse#80]

**Comment by rangaryan6** [comment]
@n1ght4ngel19 Can I help by playing your builds and bug identification?

**Comment by n1ght4ngel19** [comment]
Hey @rangaryan6! Sorry for not getting to you back sooner, life's a bit busy these days. As it stands right now, I'm not sure I'll continue this implementation as it's all over the place, and the structure is quite hard to understand after a few weeks even for me as it is.

BUT, I am making a new one with proper design choices in mind this time, and I'll hopefully have a project with similar progress in a few weeks, which you'll be able to test.

**Comment by n1ght4ngel19** [comment]
Yeah, and just to answer your question as well, I welcome testers, I'd just not recommend playing this one, but I'll put a notice in the Gruntz Universe Discord server (if you're not familiar with it, here's a link to join: https://discord.gg/pyvu7V5E It's a community of the original game's fans. Thanks for the udnerstanding!

**Comment by rangaryan6** [comment]
Hi Levente,

Thanks for getting back to me at all. Very exciting that you're continuing with the project in sooner capacity. Will you still be using the Unity platform in your next iteration?

Thanks,
Ryan

Warm regards,
Ryan Moore
<https://aka.ms/AAb9ysg>
________________________________

**Comment by rangaryan6** [comment]
Hi Levente,

Would you happen to know who made the disgruntled.com game website? I was enjoying playing it but it appears that their domain has expired.

Thanks heaps,
Ryan Moore



From: Ryan Moore ***@***.***>
Date: Thursday, 3 October 2024 at 5:30 am
To: n1ght4ngel19/gruntz-unityverse ***@***.***>, n1ght4ngel19/gruntz-unityverse ***@***.***>
Cc: Mention ***@***.***>
Subject: Re: [n1ght4ngel19/gruntz-unityverse] Breaking Giant Rockz (Issue #80)
Hi Levente,

Thanks for getting back to me at all. Very exciting that you're continuing with the project in sooner capacity. Will you still be using the Unity platform in your next iteration?

Thanks,
Ryan

Warm regards,
Ryan Moore
________________________________
